app/models/allyship.rb

```diff
@@ -12,6 +12,14 @@
 #
 
 class Allyship < ApplicationRecord
+  DISPLAY_ATTRIBUTES = %w[
```

Comment:
Let's rename this to something like `USER_DATA_ATTRIBUTES` so that it's more obvious that this is being used for that.